Sources/NDT7Settings.swift

```diff
@@ -32,19 +23,35 @@ public struct NDT7Settings {
 
     /// Define all the headers needed for NDT7 request.
     public let headers: [String: String]
+    
+    public var currentServer: NDT7Server? {
+        guard let selectedIndex = currentServerIndex else {
+            return nil
+        }
+        
+        guard selectedIndex < allServers.count else {
+            return nil
+        }
```

Comment:
Nit: You can join both in the same guard.

`guard let selectedIndex = currentServerIndex,  selectedIndex < allServers.count ...`